tests/award.js

```diff
@@ -377,7 +376,6 @@ module.exports = {
                             .end();
                         done();
                     })
-                done();
```

Comment:
I think we need to revisit this test Since the .perform functions are all async in this case, things are completing before the test is finished when done() is called.